Content.Server.Database/EmberfallModel.cs

```diff
@@ -8,11 +8,8 @@ public class EmberfallProfile
     {
         [Column("emberfallprofile_id")]
         public int Id { get; set; }
-
-        [Column("profile_id")]
-
         public int ProfileId { get; set; }
-        public Profile Profile { get; set; } = null!;
-        public string CustomSpeciesName { get; set; } = string.Empty;
+        public virtual Profile Profile { get; set; } = null!;
+        public string? CustomSpeciesName { get; set; } = string.Empty;
```

Comment:
probably want this to be null instead of string.Empty?

Reply:
riiight